lib/carwash/scrubber.rb

```diff
@@ -88,7 +88,7 @@ def obscure_sensitive_values(line, obscure_with: self.obscure_with)
   # detected. Makes replacements in place.
   def obscure_sensitive_values!(line, obscure_with: self.obscure_with)
     @sensitive_vals.each do |val|
-      line.gsub!(val, obscure_with)
+      line.gsub!(/#{val}/i, obscure_with)   
```

Comment:
`val` will need to be escaped first, to avoid special characters being interpreted as regex operators:

```
> val = ".*"
=> ".*"
> /#{val}/i
=> /.*/i
```

Reply:
Can you add a test case for #1 as well?

Reply:
I will try to add a test tonight